integration/nwo/orderer_template.go

```diff
@@ -121,6 +121,17 @@ Metrics:
     {{- end }}
     WriteInterval: 5s
     Prefix: {{ ReplaceAll (ToLower Orderer.ID) "." "_" }}
+Admin:
+  ListenAddress: 127.0.0.1:{{ .OrdererPort Orderer "Admin" }}
+  TLS:
+    Enabled: {{ .TLSEnabled }}
+    PrivateKey: {{ $w.OrdererLocalTLSDir Orderer }}/server.key
+    Certificate: {{ $w.OrdererLocalTLSDir Orderer }}/server.crt
```

Comment:
Does this (using `OrdererLocalTLSDir`) mean that in all the tests we share the crypto material of the Operations and Admin endpoints?
Is this what we want? 
Is there a testing benefit for separating `OrdererLocalTLSDir` from say, `OrdererAdminTLSDir`?